integration-tests/integration/coordinator/remove-involved-staff.spec.js

```diff
@@ -106,6 +105,10 @@ context('A use of force coordinator can remove involved staff', () => {
           { username: 'TEST_USER name', link: 'View statement', isOverdue: false, isUnverified: false },
         ])
       )
+
+    notCompletedIncidentsPage.getCountOfNotCompleteReports().then(count => {
+      expect(parseInt(count.rows[0].report_count, 10)).to.equal(0)
```

Comment:
would be good to push `parseInt(count.rows[0].report_count, 10))` into the task, as then you don't have to spread implementation details to everywhere that calls it.

Reply:
done